packages/api-watch/api-watch.0.1.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "Libraries and tools to keep watch on your OCaml lib's API changes"
+maintainer: ["Nathan Rebours <nathan.p.rebours@gmail.com>"]
+authors: ["Nathan Rebours <nathan.p.rebours@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/NathanReb/ocaml-api-watch"
+bug-reports: "https://github.com/NathanReb/ocaml-api-watch/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "5.2.0" & < "5.3.0"}
+  "ppx_expect" {with-test}
+  "ppx_deriving"
+  "logs"
+  "containers"
+  "fmt"
+  "cmdliner" {>= "1.1.0"}
+  "diffutils"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/NathanReb/ocaml-api-watch.git"
+pin-depends: [
```

Comment:
This pin-depends has to go before it can be merged.

Reply:
oh you're right! I completely forgot about it!